packages/frontend/src/Build.svelte

```diff
@@ -373,7 +373,7 @@ $: if (availableArchitectures) {
             <div class="mb-2">
               <span class="text-md font-semibold mb-2 block">Disk image type</span>
               <div class="flex items-center mb-3">
-                <label for="raw" class="ml-1 flex items-center cursor-pointer">
+                <label for="raw" class="ml-1 flex items-center cursor-pointer" aria-label="raw-checkbox">
```

Comment:
But this isn't podman desktop, as has been pointed out many times and in addition all the other aria-labels already defined in the bootc extension use this pattern, for example for the output path the field is defined as `aria-label="folder-select"` or the image selection dropdown is `aria-label="image-select"`.

I dont have a preference for the pattern personally, but I do think we shouldn't use two patterns in the same component, so I just used the pattern that was already in use in bootc.

Reply:
I totally understand. Right approach would be to have it unified in any case. No need to change existing labels in this PR. But I would like not to add new cases which we would like to have rewritten later. Your call.

Reply:
I've created https://github.com/containers/podman-desktop-extension-bootc/issues/488 to track standardization of aria-labels in bootc extension to match the pattern in PD as a separate work stream.

I'm going to leave the labels in this PR as they are currently since they match the current pattern in this extension.